app/main.py

```diff
@@ -1,3 +1,63 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: any) -> object:
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+
+    def __iadd__(self, other: any) -> callable:
+        if isinstance(other, (int, float)):
+            self.km = self.km + other
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+        return self
+
+    def __mul__(self, other: (int, float)) -> [object, None]:
+        if isinstance(other, (int, float)):
+            self.km = self.km * other
+            return self
+
+    def __truediv__(self, other: [int, float]) -> object:
+        if isinstance(other, (int, float)) and other:
+            divide = round(self.km / other, 2)
+            return Distance(divide)
+
+    def __lt__(self, other: [int, float, object]) -> bool:
```

Comment:
The type hint `[int, float, object]` is not correct. It should be `Union[int, float, Distance]` to properly indicate the types that `other` can be.